src/app/core/view/dynamic-components/fa-dynamic-icons.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  faChartLine,
+  faChild,
+  faHome,
+  faTable,
+  faUniversity,
+  faQuestionCircle,
+  faWrench,
+} from "@fortawesome/free-solid-svg-icons";
+import {
+  faCalendar,
+  faCalendarCheck,
+  faFileAlt,
+  faUser,
+} from "@fortawesome/free-regular-svg-icons";
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+
+/**
+ * font-awesome icons decoupled from the config
+ */
+export const faDynamicIcons = new Map<string, IconDefinition>([
+  ["home", faHome],
+  ["child", faChild],
+  ["university", faUniversity],
+  ["calendar", faCalendar],
+  ["calendar-check", faCalendarCheck],
+  ["table", faTable],
+  ["notes", faFileAlt],
+  ["wrench", faWrench],
+  ["user", faUser],
+  ["question", faQuestionCircle],
+  ["line-chart", faChartLine],
```

Comment:
I am not sure if this is a desirable approach. This would mean whenever we want to use a new icon (e.g. a football when setting up a instance for a soccer club) we first have to make code changes and create a release for it. What was the reason to add this layer of abstraction? 
Maybe a solution would be to have a fallback handling where if no alias can be found in this map, the `IconDefinition` is fetched from font awesome using the exact name from the config? So we would have the chance to define aliases but at the same time don't have to make code changes to use new icons.

Reply:
There is definitively room for discussion here. From my perspective, having 'hardcoded' icons in the config means that whenever the icon definitions change (as was the case when upgrading to the new font awesome icon set; e.g. there are no longer icons with an `-o` suffix) the config also has to be updated in order to reflect these changes. In my thinking, this means that we have to redefine the whole config for each instance when switching from one icon set to another (or, as in this case, from one version to another).
The other thing is that for the new font awesome version, icons can also prefixed with either `far` or `fas` indicating either regular or solid icons.

Reply:
Good point so it's either that or a little config migration script. Let's figure this out tomorrow.